bundles/org.openhab.binding.networkupstools/src/main/resources/ESH-INF/thing/dynamic-channels.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file contains spaces instead of tabs :-)

Reply:
I plead "not guilty":  https://github.com/openhab/static-code-analysis/pull/357